test/integration/runner_common.cpp

```diff
@@ -119,6 +121,48 @@ int IntegrationBase::exit_code(Proc &proc) {
   return proc.proc.exit_code();
 }
 
+fs::path IntegrationBase::createCustomConfig(std::string extraConfig) {
+  boost::trim(extraConfig);
```

Comment:
Is toml++ not ok with whitespace?

Reply:
The reason this was initially implemented was added whitespace when providing the extra config. I resolved this so this was removed.